### DISCUSSION TITLE
Unable to run after updating from 1.5.1 to 1.6.0

### DISCUSSION THREAD

**Discussion by smokey007** [garethgeorge/backrest#537] [Q&A]
Hi, when I update I keep getting an error in the logs.

```
2024-11-02T22:58:12.806-0300	INFO	writing logs to: /data/processlogs
2024-11-02T22:58:22.708-0300	WARN	operation log may be corrupted, if errors recur delete the file "/data/oplog.sqlite" and restart. Your backups stored in your repos are safe.
2024-11-02T22:58:22.709-0300	FATAL	error creating oplog: open sqlite pool: sqlite: open "/data/oplog.sqlite": enable wal: sqlite: step: database is locked
```

I have deleted it several times but it keeps running into this. I even created a brand new stack and so there was no updating but keep running into the same problem.

I looked at the enable wal: but couldn't find where to enable it in the docker config. Any help is appreciated

**Comment by garethgeorge** [comment]
1.6.0 has a bug in the migration code that assumes a previous install existed -- please run 1.6.1 (latest version) for a new install.

**Reply by smokey007** [reply]
I just tried a new install as well as updating my main instance but I am still getting the same error.

**Reply by garethgeorge** [reply]
Fixing the formatting on your error message I believe I misread it the first time. The locking you're seeing is an error coming from sqlite -- I believe it should only happen if another process is already accessing the database -- but I'm not sure how that would be happening as backrest is supposed to take an advisory lock on the db and should be giving a different message.

Did you also delete the .db-shm and .db-wal files when attempting the reset?

What platform are you running on?

**Reply by smokey007** [reply]
When I start a new instance there are no  .db-shm and .db-wal files in the data folder or cache. I am using docker.

**Reply by garethgeorge** [reply]
Is the database location properly configured and writeable for backrest? Looks like this may be a poorly formatted IO error as backrest is failing to open the DB in read/write mode

**Reply by smokey007** [reply]
The folder where it is, is writable. It does write a few files in it.
